src/components/MemberProfileEdit.tsx

```diff
@@ -39,193 +78,312 @@ const MemberProfileEdit: React.FC<MemberProfileEditProps> = ({
+                </HStack>
+              </FormLabel>
+              <Input
+                name="major"
+                value={profile.major || ''}
+                onChange={handleChange}
+                placeholder="Enter your major"
+              />
+            </FormControl>
+            <FormControl>
+              <FormLabel>
+                <HStack spacing={2}>
+                  <Icon as={FaCalendar} color={iconColor} />
+                  <Text>Graduation Date</Text>
+                </HStack>
+              </FormLabel>
+              <Input
+                name="gradDate"
+                type="date"
+                value={profile.gradDate || ''}
+                onChange={handleChange}
+              />
+            </FormControl>
+          </SimpleGrid>
+        </FormSection>
+      </Box>
+
+      <Divider />
+
+      <Box px={8} py={6} bg={sectionBg}>
+        <FormSection title="Additional Information">
+          <VStack spacing={6}>
+            <FormControl>
+              <FormLabel>
+                <HStack spacing={2}>
+                  <Icon as={FaMapMarkerAlt} color={iconColor} />
+                  <Text>Location</Text>
+                </HStack>
+              </FormLabel>
+              <Input
+                name="local"
+                value={profile.local || ''}
+                onChange={handleChange}
+                placeholder="Enter your location"
+              />
+            </FormControl>
+            <FormControl>
+              <FormLabel>Bio</FormLabel>
+              <Input
+                name="bio"
+                value={profile.bio || ''}
+                onChange={handleChange}
+                placeholder="Tell us about yourself"
+              />
+            </FormControl>
+          </VStack>
+        </FormSection>
+      </Box>
+
+      <Divider />
+
+      {/* socials */}
+      <Box px={8} py={6}>
+        <FormSection title="Social Links">
+          <VStack spacing={6}>
+            <FormControl>
+              <FormLabel>
+                <HStack spacing={2}>
+                  <Icon as={FaLinkedin} color={iconColor} />
+                  <Text>LinkedIn</Text>
+                </HStack>
+              </FormLabel>
+              <HStack spacing={2}>
+                <Input
+                  name="linkedin.username"
+                  value={profile.linkedin?.username || ''}
+                  onChange={handleSocialFieldChange}
+                  placeholder="Enter your LinkedIn URL"
+                />
+                <Tooltip
+                  label={
+                    profile.linkedin?.username
+                      ? 'Toggle visibility'
+                      : 'Add username first'
+                  }
+                >
+                  <Switch
+                    isDisabled={!profile.linkedin?.username}
+                    isChecked={profile.linkedin?.isPrivate}
+                    onChange={() => handleSocialFieldToggle('linkedin')}
```

Comment:
Does this still return the information from the backend even if it's private? Like can someone see iti n the network call? If so this is pointless and I'd just remove the isprivate completely.

Reply:
yes through directory i am pretty sure it does